backend/src/main/java/com/chingu/ChinguBoard/config/SecurityConfig.java

```diff
@@ -53,8 +54,8 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .cors(withDefaults()) // withDefaults() uses Bean with name CorsConfigurationSource
                 .csrf(csrf -> csrf.disable()) // change for deploy
                 .authorizeHttpRequests(auth -> auth
-                        .requestMatchers("/api/auth/**").permitAll()
-                        .anyRequest().authenticated())
+                        // .requestMatchers("/api/auth/**").permitAll()
```

Comment:
can you change this back to the original?
.requestMatchers("/api/auth/**").permitAll()
 .anyRequest().authenticated())